src/components/Modal/VaultStrategyModal/VaultStrategyModal.tsx

```diff
@@ -37,8 +37,8 @@ export const VaultStrategyModal: FC<
   const { chain } = useNetwork()
 
   const strategyTextAddress = isCompounderProduct
-    ? vaultProps.asset
-    : vaultProps.vaultAddress
+    ? `compounder_${vaultProps.asset}`
```

Comment:
This is a bit of a "magic number" pattern; prone to errors and typos. It would be cleaner to structure the `strategyText` object differently:

```ts
const strategyText = {
  compounder: {
    "0x...": ()
  },
  concentrator: {
    "0x...": ()
  }
}

// accessing it
const text =
  strategyText[productType][
    vaultProps[isCompounderProduct ? "asset" : "vaultAddress"]
  ]
```

Reply:
Agreed